backend/src/sms/services/sms.service.ts

```diff
@@ -82,10 +88,13 @@ const sendMessage = (
     throw new InvalidRecipientError('Invalid phone number')
   }
 
-  const client = config.get('smsFallback.activate')
-    ? new SnsSmsClient()
-    : new TwilioClient(credential)
-  return client.send(recipient, message)
+  if (config.get('smsFallback.activate')) {
+    const client = new SnsSmsClient()
```

Comment:
I actually think we should just deprecate Sns fallback. Based on my observations from the last few incidents, we didn't even activate this option at all. Planning to include this deprecation as part of the refactor but it could be started here as well.
Lmk if you got a differing opinion @tan-zx

Reply:
done